x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/action_list_date_range_picker.tsx

```diff
@@ -5,33 +5,32 @@
  * 2.0.
  */
 
-import React, { memo, useState, useMemo } from 'react';
-import dateMath from '@kbn/datemath';
-import { EuiSuperDatePicker } from '@elastic/eui';
+import React, { useCallback, memo, useState } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiSuperDatePicker, EuiSuperUpdateButton } from '@elastic/eui';
+import type { IDataPluginServices } from '@kbn/data-plugin/public';
 import { euiStyled } from '@kbn/kibana-react-plugin/common';
 import type { EuiSuperDatePickerRecentRange } from '@elastic/eui';
 import { useKibana } from '@kbn/kibana-react-plugin/public';
-import { DurationRange, OnRefreshChangeProps } from '@elastic/eui/src/components/date_picker/types';
-import { useUiSetting$ } from '../../../../common/lib/kibana';
-import { DEFAULT_TIMEPICKER_QUICK_RANGES } from '../../../../../common/constants';
+import type {
+  DurationRange,
+  OnRefreshChangeProps,
+} from '@elastic/eui/src/components/date_picker/types';
+import { UI_SETTINGS } from '@kbn/data-plugin/common';
+import type { useGetEndpointActionList } from '../../../hooks';
+import { useTestIdGenerator } from '../../../hooks/use_test_id_generator';
 
 export interface DateRangePickerValues {
   autoRefreshOptions: {
     enabled: boolean;
     duration: number;
   };
-  startDate?: string;
-  endDate?: string;
+  startDate: string;
+  endDate: string;
   recentlyUsedDateRanges: EuiSuperDatePickerRecentRange[];
 }
-interface Range {
-  from: string;
-  to: string;
-  display: string;
-}
 
 const DatePickerWrapper = euiStyled.div`
-  width: ${(props) => props.theme.eui.fractions.single.percentage};
+  width: ${(props) => props.theme.eui.fractions.fourths.percentage};
```

Comment:
**note:** This width property will go away when the additional filters are added.

Reply:
Here's an updated screenshot with full-width date picker input. @kevinlog 
![Screenshot 2022-07-05 at 15 59 55](https://user-images.githubusercontent.com/1849116/177345410-04992e0a-f214-4c3f-92ca-b6ee6827f977.png)